src/ledger/LedgerTxn.cpp

```diff
@@ -2684,6 +2733,12 @@ LedgerTxnRoot::Impl::tableFromLedgerEntryType(LedgerEntryType let)
         return "claimablebalance";
     case LIQUIDITY_POOL:
         return "liquiditypool";
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
+    case CONTRACT_DATA:
+        return "contractdata";
+    case CONFIG:
```

Comment:
`CONFIG_ENTRY` isn't great since we'd wind up with `ConfigEntryEntry`. In @sisuresh's earlier iterations of this he had it named with a mix of "config" and "configsetting". If you like I can change it to `CONFIG_SETTING` and therefore `ConfigSettingEntry` etc.

Reply:
Changed to CONFIG_SETTING